test/types/file/bradc/scalar/floatcomplexexceptions-aarch64.compopts

```diff
@@ -0,0 +1,2 @@
+--edition=2.0 # floatcomplexexceptions.old.good
+--edition=preview # floatcomplexexceptions-aarch64.good
```

Comment:
`darwin` is separated with `.`, this file is with `-`. Consistent naming would be better, I think.

Reply:
Darwin is separated with a `.` because sub_test can actually pick it up automatically.  It can't do the same with `aarch64`, or at least it didn't look like it could based on reading the relevant portion of sub_test

Reply:
Oh, OK. Didn't get that detail before.